package.json

```diff
@@ -1,52 +1,62 @@
 {
-  "name": "webpack-react-tutorial",
-  "version": "1.0.0",
-  "description": "",
-  "homepage": "https://taok-monitor.github.io/taok-front",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "serve": "cross-env NODE_ENV=development webpack-dev-server --mode development",
-    "build": "webpack --mode production",
-    "deploy": "gh-pages -d build"
-  },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
-  "devDependencies": {
-    "@babel/core": "^7.2.2",
-    "@babel/preset-env": "^7.3.1",
-    "@babel/preset-react": "^7.0.0",
-    "axios": "^0.19.0",
-    "babel-loader": "^8.0.5",
-    "babel-preset-react": "^6.24.1",
-    "bootstrap": "^4.3.1",
-    "chart.js": "^2.7.3",
-    "clean-webpack-plugin": "^1.0.1",
-    "cross-env": "^5.2.0",
-    "css-loader": "^2.1.0",
-    "file-loader": "^4.0.0",
-    "gh-pages": "^2.0.1",
-    "html-loader": "^0.5.5",
-    "html-webpack-plugin": "^3.2.0",
-    "http-signature": "^1.2.0",
-    "mini-css-extract-plugin": "^0.5.0",
-    "node-sass": "^4.11.0",
-    "optimize-css-assets-webpack-plugin": "^5.0.1",
-    "prop-types": "^15.6.2",
-    "raw-loader": "^1.0.0",
-    "react": "^16.8.1",
-    "react-chartjs-2": "^2.7.4",
-    "react-dom": "^16.8.1",
-    "reactstrap": "^7.1.0",
-    "sass-loader": "^7.1.0",
-    "style-loader": "^0.23.1",
-    "webpack": "^4.29.3",
-    "webpack-cli": "^3.2.3",
-    "webpack-dev-server": "^3.1.14"
-  },
-  "dependencies": {
-    "react-datepicker": "^2.4.0",
-    "tar": ">=4.4.2"
-  }
+	"name": "webpack-react-tutorial",
```

Comment:
@samuelsilvadev you can change this name, maybe 'taok-front'

Reply:
perfect!!